language/de/install.php

```diff
@@ -180,18 +181,17 @@
 	'DB_PASSWORD'			=> 'Datenbank-Passwort',
 	'DB_NAME'				=> 'Name der Datenbank',
 	'DB_USERNAME'			=> 'Datenbank-Benutzername',
+	'DATABASE_VERSION'		=> 'Databankversion',
```

Comment:
Da alles andere mit Bindestrich abgetrennt ist würde ich das hier auch so machen (aber ersetz' auf jeden Fall auch das Data durch Daten ;) ), auch wenn zusammenschreiben in allen Fällen vermutlich richtiger wäre ist so der Unterschied der Textfelder einfacher zu erkennen.